dashboard/test/ui/README.md

```diff
@@ -100,6 +102,7 @@ Run **eyes tests** on one feature in one saucelabs browser against your local ma
 
 - If you're new to [Cucumber](http://cukes.info/), read about [Cucumber scenarios](https://github.com/cucumber/cucumber/wiki/Feature-Introduction), especially the keywords [Given When Then](https://github.com/cucumber/cucumber/wiki/Given-When-Then).
 - When debugging test scripts, it can be helpful to add pauses, such as: `And I wait for 5 seconds`.
+- If you're missing data locally, try running `bundle exec rake seed:ui_test` from the dashboard directory
```

Comment:
This is the only change to the file, the rest of the changes were auto-formatting on save